src/components/menu-left/MenuLeft.vue

```diff
@@ -123,6 +130,19 @@ export default Vue.extend({
     },
   },
   methods: {
+    isWarning(index: number): boolean {
+      if (
+        this.$store.state.show.stuntSheets[index].warnings.length !== 0 ||
+        this.$store.state.show.warnings.some((warning: Warning) => {
+          return warning.stuntSheets.some((ss: number) => {
+            return ss === index;
+          });
+        })
+      ) {
+        return true;
+      }
+      return false;
```

Comment:
Could we simplify this to just return the conditional? Will help with readability!
```suggestion
      return this.$store.state.show.stuntSheets[index].warnings.length !== 0 ||
        this.$store.state.show.warnings.some((warning: Warning) => {
          return warning.stuntSheets.some((ss: number) => {
            return ss === index;
          });
        })
```